code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -292,7 +292,7 @@
 	pack.reagents.add_reagent("tricordrazine", 15 * pack.storage_slots)
 	pack.desc += " 'T' has been scribbled on it."
 
-	new /obj/item/flame/lighter/zippo(src)
+	new /obj/item/flame/lighter/zippo/excelsior(src)
```

Comment:
These are not excelsior smokes

Reply:
Can you suggest the change so I can push it or should I make a commit then? Should it be a solfed one? Standard one? Black one?